### DISCUSSION TITLE
AUR Package

### DISCUSSION THREAD

**Discussion by n194** [cupcakearmy/autorestic#66] [General]
I wrote an AUR package for autorestic for any Arch users here :)

[aur/autorestic-bin](https://aur.archlinux.org/packages/autorestic-bin/)

**Comment by cupcakearmy** [comment]
Awesome stuff! I've added it to the install guide. https://autorestic.vercel.app/installation
I am not an expert on writing AUR packages, do they update automatically or do you have to maunally version bump?

**Reply by n194** [reply]
[The PKGBUILD](https://aur.archlinux.org/cgit/aur.git/tree/PKGBUILD?h=autorestic-bin) has to be manually updated but usually it's as simple as bumping the version number (pkgver).
Edit: the checksums also have to be updated.

**Reply by cupcakearmy** [reply]
I see, thanks for your work :)
